LICENSE

```diff
@@ -175,18 +176,7 @@ Apache License
 
    END OF TERMS AND CONDITIONS
 
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2019 Red Hat, Inc.
```

Comment:
2021?

Reply:
yes! I never know if it's the starting date or end date. I changed to 2021, Thanks.

Reply:
I never know either in doubt, put both!

Reply:
A quick look seems to say it's the starting date which matters ( https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year https://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/ )